packages/widgetbook_generator/README.md

```diff
@@ -10,60 +10,4 @@
 
```

Comment:
Widgetbook is a flutter package which helps developers cataloguing their widgets, testing them quickly on multiple devices and themes, and sharing them easily with designers and clients. Inspired by Storybook.js.